config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -355,39 +355,6 @@ config MBEDTLS_GCM_C
 config MBEDTLS_RSA_C
     default n
 
-config PSA_WANT_KEY_TYPE_CHACHA20
```

Comment:
Does it mean we now should explicitly enable all crypto algorithms used by Matter?

Reply:
That is the goal. But for now only on the PSA configurations.
The mbedtls ones are not touched in this.